docker-cpp-demos/rtsp-docker/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM ubuntu:18.04
+
+RUN apt-get upgrade && \
+    apt-get update && \
+    apt-get install -y  \
+    byacc \
+    cmake \
+    curl \
+    g++ \
+    git \
+    gstreamer1.0-plugins-base-apps \
+    gstreamer1.0-plugins-bad \
+    gstreamer1.0-plugins-good \
+    gstreamer1.0-plugins-ugly \
+    libgstreamer1.0-dev \
+    libgstreamer-plugins-base1.0-dev \
+    m4 \
+    maven \
+    openjdk-8-jdk \
+    python2.7 \
+    pkg-config \
+    vim \
+    wget  \
+    xz-utils && \
+    rm -rf /var/lib/apt/lists/* && \
+    cd /opt/
+
+# ===== Setup Kinesis Video Streams Producer SDK (CPP) =======================================
+WORKDIR /opt/
+RUN git clone https://github.com/awslabs/amazon-kinesis-video-streams-producer-sdk-cpp.git
+WORKDIR /opt/amazon-kinesis-video-streams-producer-sdk-cpp/
+RUN git submodule update --init
+WORKDIR /opt/amazon-kinesis-video-streams-producer-sdk-cpp/build/ && \
+RUN cmake .. -DBUILD_GSTREAMER_PLUGIN=ON &&\
+    make && \
+    cp start_rtsp_in_docker /opt/amazon-kinesis-video-streams-producer-sdk-cpp/build/
```

Comment:
since we removed the script, this is no longer necessary.

Reply:
Not sure why it showed up. Did a soft reset and pushed the change again.